### ISSUE TITLE
Basked Custom Selection results in error

### ISSUE THREAD

**Issue by neubi4** [Icinga/icingaweb2-module-director#1733]
## Expected Behavior
Getting some form field or similar to select a subset of Commands/Host Templates/etc when choosing "Custom Selection" in the basked form.

## Current Behavior
You can only choose the select field, submitting results in error messages like `Undefined index: Command (Basket.php:148)`.

## Steps to Reproduce (for bugs)
Create a basked, choose "Custom Selection" for any type and try to submit the basked.
![director](https://user-images.githubusercontent.com/1274627/49785349-98415780-fd20-11e8-9604-1b0fad7a6516.PNG)

## Your Environment
* Director version (System - About): 1.6.0
* Icinga Web 2 version and modules (System - About): 2.6.2
* Icinga 2 version (`icinga2 --version`):
```
icinga2 - The Icinga 2 network monitoring daemon (version: r2.10.2-1)

Copyright (c) 2012-2018 Icinga Development Team (https://icinga.com/)
License GPLv2+: GNU GPL version 2 or later <http://gnu.org/licenses/gpl2.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

System information:
  Platform: CentOS Linux
  Platform version: 7 (Core)
  Kernel: Linux
  Kernel version: 3.10.0-862.14.4.el7.x86_64
  Architecture: x86_64

Build information:
  Compiler: GNU 4.8.5
  Build host: unknown

Application information:

General paths:
  Config directory: /etc/icinga2
  Data directory: /var/lib/icinga2
  Log directory: /var/log/icinga2
  Cache directory: /var/cache/icinga2
  Spool directory: /var/spool/icinga2
  Run directory: /run/icinga2

Old paths (deprecated):
  Installation root: /usr
  Sysconf directory: /etc
  Run directory (base): /run
  Local state directory: /var

Internal paths:
  Package data directory: /usr/share/icinga2
  State path: /var/lib/icinga2/icinga2.state
  Modified attributes path: /var/lib/icinga2/modified-attributes.conf
  Objects path: /var/cache/icinga2/icinga2.debug
  Vars path: /var/cache/icinga2/icinga2.vars
  PID path: /run/icinga2/icinga2.pid
```
* Operating System and version: CentOS Linux release 7.6.1810
* Webserver, PHP versions: Apache 2.4.6, PHP 7.2.12

**Comment by Thomas-Gelf** [comment]
Sorry, I broke this with 0c2e3fe when fixing another issue shortly before tagging v1.6.0. Happens only wit "Custom Selection", will be fixed immediately

**Comment by Thomas-Gelf** [comment]
Fixed, will be tagged as v1.6.1 soon. In the meantime you can use the `support/1.6` branch. The master is also fine.
